src/tests/Conversion.test.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
- [ ] Just as the previous reviewer suggested, please avoid disabling linter checks. The linters are there to help you adhere to good coding standards and ensure that the whole team is consistent with the same standards as well.